### PR TITLE
fix(chat notification): duplication of first message notification

### LINKED ISSUE #15088: [2.5] - First chat notification duplicated
**Describe the bug**
After enabling chat notification, the first notification received by the user is being duplicated.

**To Reproduce**
1. Join with 2 users;
2. As user A, go to `Settings > Notifications > Chat message` and enable `Popup Alerts`;
3. As user A, clear all notifications and hide public chat;
4. As user B, send a public message;

**Automated tests related**
- `Notifications - Chat - Public chat notification`;
- `Notifications - Chat - Private chat notification`;

**Video**

https://user-images.githubusercontent.com/56703993/170557422-15340e6b-1c99-4be1-84cd-fd74f19429c8.mp4

### PR DESCRIPTION
**Pull request by JoVictorNunes**
### What does this PR do?

Fixes duplication of the popup notification of the first chat message.

### Closes Issue(s)

Closes #15088

### Motivation

n/a

### More

n/a

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/utils/hooks/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { useEffect, useRef } from 'react';
+
+export const usePrevious = (value) => {
```
Comment (schrd) [inline bigbluebutton-html5/imports/ui/components/utils/hooks/index.js:3]: Why did you name this function `usePrevious`? When I read the code above I don't get the relationship between the function name and what it does.

Comment (JoVictorNunes) [reply bigbluebutton-html5/imports/ui/components/utils/hooks/index.js:3]: Agreed!

What do you suggest? Maybe `usePreviousValue`, `usePreviousState`?

**Review by antonbsa — APPROVED** [review]
Manual and automated tests passing!

image

**Comment by antobinary** [comment]
Does this close https://github.com/bigbluebutton/bigbluebutton/issues/15079 ?

**Comment by antonbsa** [comment]
> Does this close https://github.com/bigbluebutton/bigbluebutton/issues/15079 ?

I believe so, because as far as I could test this bug seems to be related to the UI only (which is fixed on this PR), not to the testing code.

**Pull request merged**